README.rst

```diff
@@ -3,60 +3,69 @@
 render-guides
 =============
 
-Render TYPO3 Documentation with `phpdocumentor/guides`.
+This is the documentation rendering tool for TYPO3 projects. It is based on
+`phpDocumentor/guides`_ and can be used as a drop-in replacement for Sphinx.
+The tool is used by the automated documentation rendering system of the
+TYPO3 project. And can be used by documentation authors to validate their
+documentation.
 
-Usage with Docker
-=================
+Some basic commands are listed below, for more information, see the
+`Documentation` subdirectory of this project.
+
+Usage with Docker (via supplied container)
+==========================================
 
 ::
+
+    # Create output directory.
     mkdir Documentation-GENERATED-temp
+
+    # Execute the Docker container that is provided remotely.
+    # Renders all files in the `Documentation` and store in `Documentation-GENERATED-temp`.
     docker run --rm --pull always -v $(pwd):/project -it ghcr.io/typo3-documentation/render-guides:main --output Documentation-GENERATED-temp Documentation
 
-Contributing
-============
+(see :ref:`_Setup_Docker:Docker containers` for complete documentation)
 
-When contributing please run all tests before committing::
+Usage with Docker (via custom container)
+========================================
 
-    make pre-commit-test
+::
 
-You can use a helper script to set this up once in your project::
+    # Build the custom local Docker container
+    docker build --file Dockerfile --tag typo3-docs:local .
 
-    make githooks
+    # Execute the Docker container that is provided locally, build Documentation
+    docker run --rm --user=$(id -u):$(id -g) --volume ${PWD}:/project typo3-docs:local ./Documentation ./Documentation-GENERATED-temp --theme=typo3docs
```

Comment:
please omit `--user`,  the output directory is changed to `--output` and can be omitted, as we set this by default, same applies for the input director, which is automatically detected. 

We can simplify this to:

`docker run --rm --user=$(id -u):$(id -g) --volume ${PWD}:/project --progress`

Reply:
You probably mean:

`docker run --rm --user=$(id -u):$(id -g) --volume ${PWD}:/project typo3-docs:local --progress`

(note the "typo3-docs:local" in there). However, when I run this with "--version" instead of "--progress" I still do not get the output of the interpreted --version command?!